package.json

```diff
@@ -35,14 +38,14 @@
   "devDependencies": {
     "@defi-wonderland/crypto-husky-checks": "1.3.0",
     "@testing-library/jest-dom": "6.1.2",
-    "@testing-library/react": "14.0.0",
+    "@testing-library/react": "^14.2.1",
     "@testing-library/user-event": "14.4.3",
     "@types/lodash.merge": "4.6.7",
     "@types/react": "18.2.15",
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "6.0.0",
     "@typescript-eslint/parser": "6.0.0",
-    "@vitejs/plugin-react": "4.0.3",
+    "@vitejs/plugin-react": "^4.2.1",
```

Comment:
you can now remove all vite related dependencies

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/9a9683fc0ce55ee0a3c687f7ac80783cade96b46